src/libsyncengine/reconciliation/platform_inconsistency_checker/platforminconsistencycheckerworker.cpp

```diff
@@ -40,11 +40,11 @@ void PlatformInconsistencyCheckerWorker::execute() {
     checkTree(ReplicaSide::Local);
 
     for (const auto &idItem: _idsToBeRemoved) {
-        if (!_syncPal->updateTree(ReplicaSide::Remote)->deleteNode(idItem.remoteId)) {
-            LOGW_SYNCPAL_WARN(_logger, L"Error in UpdateTree::deleteNode: node id=" << Utility::s2ws(idItem.remoteId.c_str()));
+        if (!idItem.remoteId.empty() && !_syncPal->updateTree(ReplicaSide::Remote)->deleteNode(idItem.remoteId)) {
```

Comment:
Is it possible for `idItem.remoteId` to be empty, I mean, in some natural scenario?

Reply:
Yes it is in case of Create operation. And it is verified in logs in a few RMs. Therefor, there was a warning displayed in log that could be simply ignored